### PR TITLE
chore: implement blockquote block

### PR DESCRIPTION
**Pull request by mhaack**
Test URLs:
- Before: https://main--wknd--hlxsites.hlx.live/magazine/western-australia
- After: https://quote--wknd--hlxsites.hlx.live/magazine/western-australia
- With Theme: https://quote--wknd--hlxsites.hlx.live/drafts/mhaack/test/western-australia

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -225,22 +226,14 @@ main pre {
...
 
 main blockquote {
-  font-style: italic;
+  color: var(--blockquote-color);
+  background-color: var(--blockquote-background-color);
+  font-family: var(--heading-font-family);
   margin: 3rem;
   text-indent: -1rem;
   hanging-punctuation: first;
 }
```
Comment (fkakatie) [inline styles/styles.css:235]: I would personally move this out of `styles.css` and into `blockquote.css`.

**Comment by fkakatie** [comment]
Is this first column necessary? From my perspective, this adds complexity to the authoring experience.

Screen Shot 2023-02-08 at 8 52 56 AM

File: blocks/blockquote/blockquote.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  margin: 0;
+  text-indent: 0;
+  padding: 0 32px;
+}
+
+.blockquote blockquote::after {
+  border-bottom: 2px solid var(--heading-highlight-color);
+  content: '';
+  display: block;
+  width: 5pc;
```
Comment (fkakatie) [inline blocks/blockquote/blockquote.css:16]: I don't know that I've ever seen `pc` used for web. Could this be translated to `px` like the rest of the units in the block?

**Review by fkakatie — APPROVED** [review]

**Comment by mhaack** [comment]
> Is this first column necessary? From my perspective, this adds complexity to the authoring experience.

Change the block structure. No first column, only quote text and optionally author. See https://quote--wknd--hlxsites.hlx.page/drafts/mhaack/test/western-australia

**Pull request merged**